### DISCUSSION TITLE
Dark/light button behaving differently for documentation pages

### DISCUSSION THREAD

**Discussion by vinnciane** [imfing/hextra#317] [General]
Hello, 

I have noticed that the dark/light button is displayed differently in my documentation pages, compared to the home page where it shows the icon and the text. In the doc pages, it is placed a little bit more on the right and shows no text. 
I've looked into the Hextra example site and this is happening there too, on the documentation pages. I think this is probably not something intended. 

![image](https://github.com/imfing/hextra/assets/45791691/7410373c-1e1b-4ebe-98cc-f0150c98184a)

Thank you

**Comment by imfing** [comment]
It is intended, on the docs sidebar, the label is hidden
